src/main/java/com/hackclub/hccore/slack/SlackBot.java

```diff
@@ -334,4 +454,31 @@ void sendMessage(String msg, String iconURL, String username) throws IOException
       e.printStackTrace();
     }
   }
+
+  public void sendVerificationMessage(String id, String mcName) throws IOException {
+    MethodsClient client = Slack.getInstance().methods();
```

Comment:
May not be enabled, ensure there is a fallback (like a nice message saying it's disabled)
Edit: is is in the slack class, on the instance itself.
As it's non-static, why not just use `this`

Reply:
sorry, seems i was mistaken with this, i thought that `Slack` == `SlackBot`
if prefered (as it was shorter), it can be reverted